doc/rtd/explanation/about-cloud-config.rst

```diff
@@ -0,0 +1,85 @@
+.. _about-cloud-config:
+
+About the cloud-config file
+***************************
+
+The ``#cloud-config`` file is a type of user data that cloud-init can consume
+to automatically set up various aspects of the system.
+
+It overrides all other types of user data, so if there is a conflict between
+the config you specify in other :ref:`user data formats <user_data_formats>`,
+the contents of the ``#cloud-config`` file will take precendence.
+
+Note that networks are not configurable via the ``#cloud-config`` file because
+:ref:`network configuration <network_config>` occurs before user data is
+consumed and applied.
+
+How do I create a cloud-config file?
+====================================
+
+The cloud-config file can be written in any valid YAML but the first line
+**must start** with ``#cloud-config``. This line identifies the file to
+cloud-init and ensures that it will be processed as intended.
+
+After the first line, every aspect of the system's configuration is controlled
+through specific cloud-init **modules**. Each module included in the
+``cloud-config`` file can be thought of as a section.
+
+Let us consider the example of the "keyboard" module. The module has a
+corresponding top-level key (``keyboard:``, in this case), and beneath that
+the various configuration parameters are defined:
+
+.. code-block:: yaml
+
+   keyboard:
+     layout: us
+     model: pc105
+     variant: nodeadkeys
+     options: compose:rwin
+
+A full list of modules can be found :ref:`on our modules page <modules>`. This
+list also shows the valid schema keys for every module, and YAML examples.
+
+Module ordering
+---------------
+
+The order of the different module "sections" is mostly unimportant and modules
+can be shown in any order -- except where there are dependencies. For example,
+if you want to create users and also put them in a specific group, then the
+``groups`` section must go before ``users`` so that the groups are created
+first.
+
+Cloud-config for the live installer
```

Comment:
I would replace `live` with `Ubuntu`.

I don't love that we're so prominently calling out subiquity in upstream documentation that should be distro independent. Tbh, I think buried in https://cloudinit.readthedocs.io/en/latest/reference/faq.html#autoinstall-preruncmd-postruncmd is the right place for it.

Reply:
Perhaps, but I'm trying to slim down the FAQ so I'd rather not put it there. Since it does represent a frequently asked question (or at least, something that does cause a lot of problems/consternation) I'm actually OK with putting it in the docs - although I'm also happy to have it at the end of the file so that it's less prominent, if you think that would be better?